consensus/src/consensusdb/mod.rs

```diff
@@ -18,9 +18,6 @@ use schema::{BLOCK_CF_NAME, QC_CF_NAME, SINGLE_ENTRY_CF_NAME};
 use schemadb::{ReadOptions, SchemaBatch, DB, DEFAULT_CF_NAME};
 use std::{collections::HashMap, iter::Iterator, path::Path, time::Instant};
 
-type HighestTimeoutCertificate = Vec<u8>;
-type VoteMsgData = Vec<u8>;
```

Comment:
why though? I thought this made the API a little cleaner.

Reply:
I was trying to avoid confusion since we already have VoteMsg, Vote, VoteProposal, VoteData etc.